### PR TITLE
Feature/add legal content

### PR DESCRIPTION
**Pull request by anthonygarvan**
This PR builds out the legal page in preparation for releasing eregs as described in 18F/fec-eregs#68 including
- integrating @emileighoutlaw 's content
- removing advanced search (from async discussion with @edmullen 
- restricting search criteria to Regulations only
- adding template as described in issue and refined discussions in #fec-design, which clarified that due to css issues it would be better to have message above content rather than in right pane (where there was an offset). 
- links added for ethnio and feedback

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
Content looks good to me! Thanks!

File: fec/legal/templates/legal/home.html [hunk 1]
```diff
@@ -6,6 +6,67 @@
 
 {% block content %}
   {% include "partials/search-hero.html" %}
+  <div class="container">
+      <div class="message message--no-icon">We're designing a new way to get legal resources.
```
Comment (adborden) [inline fec/legal/templates/legal/home.html]: Use smart quotes plz.

File: fec/legal/templates/legal/home.html [hunk 2]
```diff
@@ -6,6 +6,67 @@
...
+        We could use your help! If you're interested in <a href="javascript:void(0);"
+        onclick="feedbackWidget.show(); return false;">sharing feedback</a> or
+        <a href="javascript:void(0);" onclick="Ethnio.show(); return false;">
```
Comment (adborden) [inline fec/legal/templates/legal/home.html]: Eventually I think we'll want this in a page specific `legal.js` file, but I'm okay with this for now :)

File: fec/legal/templates/legal/home.html [hunk 3]
```diff
@@ -6,6 +6,67 @@
 
 {% block content %}
   {% include "partials/search-hero.html" %}
+  <div class="container">
+      <div class="message message--no-icon">We're designing a new way to get legal resources.
+        We could use your help! If you're interested in <a href="javascript:void(0);"
```
Comment (adborden) [inline fec/legal/templates/legal/home.html]: Smart quotes here too.

File: fec/legal/templates/legal/home.html (hunk 1 above)
Comment (adborden) [inline fec/legal/templates/legal/home.html]: &rsquo;

**Pull request merged**